README.md

```diff
@@ -1,561 +1,53 @@
-[![CI](https://github.com/f3d-app/f3d/actions/workflows/ci.yml/badge.svg)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Downloads](https://img.shields.io/reddit/subreddit-subscribers/f3d_app.svg)](https://www.reddit.com/r/f3d_app)
+---
+layout: default
+title: Home
+nav_order: 0
+description: "F3D - Fast and minimalist 3D viewer"
+permalink: /
+---
 
-![F3D Logo](./resources/logo.svg)
+[![CI](https://github.com/f3d-app/f3d/actions/workflows/ci.yml/badge.svg)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Downloads](https://img.shields.io/reddit/subreddit-subscribers/f3d_app.svg)](https://www.reddit.com/r/f3d_app)
 
 # F3D - Fast and minimalist 3D viewer
-
 By Michael Migliore and Mathieu Westphal.
 
-F3D (pronounced `/fɛd/`) is a [VTK-based](https://vtk.org) 3D viewer following the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle), so it is minimalist, efficient, has no GUI, has simple interaction mechanisms and is fully controllable using arguments on the command line.
+<img src="https://user-images.githubusercontent.com/3129530/195972584-11ebb000-4939-47f4-9719-9d55e18867b0.png" align="left" width="20px"/>
+F3D (pronounced `/fɛd/`) is a fast and minimalist 3D viewer. It supports many file formats, from digital content to scientific datasets (including glTF, STL, STEP, PLY, OBJ, FBX, Alembic), can show animations and support lot of rendering and texturing options including real time physically based rendering and raytracing.
+<br clear="left"/>
 
-F3D is open-source and cross-platform (tested on Windows, Linux and macOS).
-It supports a range of file formats (including animated glTF, STL, STEP, PLY, OBJ, FBX), and provides numerous rendering and texturing options.
+It is fully controllable from the command line and support configuration files. It can provide thumbnails and support interactive hotkeys.
 
-<img src="https://f3d-app.github.io/f3d/gallery/04-f3d.png" width="640">
+F3D also contains the libf3d, a simple library to render meshes, with C++, Python and Java Bindings.
+
+<img src="https://user-images.githubusercontent.com/3129530/194735416-3f386437-456c-4145-9b5e-6bb6451d7e9a.png" width="640">
 
 *A typical render by F3D*
 
-<img src="https://f3d-app.github.io/f3d/gallery/dota.gif" width="640">
+<img src="https://user-images.githubusercontent.com/3129530/194735261-dd6f1c1c-fa57-47b0-9d27-f735d18ccd5e.gif" width="640">
```

Comment:
> Can we replace the animated gif on the home page with a video?

@Meakk 

We can but videos are not autoplay on github markdown. You can take a look at the:
 - readme : https://github.com/mwestphal/f3d/blob/jekyll_doc/README.md
 - gallery: https://github.com/mwestphal/f3d/blob/jekyll_doc/GALLERY.md
 
 to see the difference. I think having a moving gif in the main readme of the project is important enough.

Reply:
> I think having a moving gif in the main readme of the project is important enough.

I agree but on the other hand, the quality isn't great. I'm fine with keeping the gif or replace it by a video, it's up to you.